ZelBack/src/services/appsService.js

```diff
@@ -10362,10 +10417,13 @@ async function syncthingApps() {
                 if (cache.numberOfExecutions === cache.numberOfExecutionsRequired) {
                   syncthingFolder.type = 'sendreceive';
                 } else if (cache.numberOfExecutions === cache.numberOfExecutionsRequired + 1) {
-                  log.info(`SyncthingApps starting appIdentifier ${appId}`);
+                  log.info(`SyncthingApps changing syncthing type to sendreceive for appIdentifier ${appId}`);
                   syncthingFolder.type = 'sendreceive';
-                  // eslint-disable-next-line no-await-in-loop
-                  await appDockerRestart(id);
+                  if (containerDataFlags.includes('r')) {
```

Comment:
only for r?

Reply:
yeah the restart for g: is done on masterSlaveApps
but thank you for pointing it out as that leads to a bug bug, all those includes should had like r: and not only r, same for s: and g:

Reply:
false alarm, all is good